src/js/tech/tech.js

```diff
@@ -322,6 +322,7 @@ class Tech extends Component {
     }
 
     let textTracksChanges = Fn.bind(this, function() {
+      let tracks = this.textTracks();
       let updateDisplay = () => this.trigger('texttrackchange');
```

Comment:
Would better as another PR if you want. Otherwise, we'll fix it at some point :)